### ISSUE TITLE
/prove-identity - copy edits required

### ISSUE THREAD

**Issue by EUzkuraityte** [alphagov/verify-local-patterns#740]
Lizzie's comments: 
1. Remove the bold copy line "How do you want us to check your identity?"
2. Please follow the prototype which includes bulleted explanations of the 2 options. If this isn't technically possible please advise up and we can come up with a best practice and relevant solution together.

![screen shot 2017-10-23 at 12 30 33](https://user-images.githubusercontent.com/20285115/31886874-056ee328-b7ee-11e7-9088-f67cf1300d0f.png)

**Comment by joanneSouthern** [comment]
Yes, its been raised with them, but no fix yet. As a compromise, I can put bullet points underneath or as a button so the user can toggle between showing and hiding the information accordingly

**Comment by EUzkuraityte** [comment]
Hi Joanne, if we could replicate the solution done for bus passes, that'd be sufficient

**Comment by joanneSouthern** [comment]
done in test
